tests/test_writer.py

```diff
@@ -1386,6 +1386,16 @@ def test_new_removes():
     assert b"Chap" not in bb
     assert b" TJ" not in bb
 
+    writer = PdfWriter()
+    writer.clone_document_from_reader(reader)
+    writer.remove_text(font_names=["LiberationSans-Bold"])
+    b = BytesIO()
+    writer.write(b)
+    reader = PdfReader(b)
+    text = reader.pages[0].extract_text()
+    assert "Arbeitsschritt" not in text
```

Comment:
For the negative case, I would recommend to add an assertion before removing the text as well.

Reply:
Thanks! Done.